src/rate_limiter.rs

```diff
@@ -32,8 +33,11 @@ use std::time::Instant;
 const CAPACITY: u64 = 20 * 1024 * 1024;
 /// The number of bytes per second the `RateLimiter` will "leak".
 const RATE: f64 = 20.0 * 1024.0 * 1024.0;
-/// Charge (in bytes) for a client get request.
-const CLIENT_GET_CHARGE: u64 = 2 * 1024 * 1024;
+
+const GET_MUTABLE_DATA_SHELL_CHARGE: u64 = LIST_MUTABLE_DATA_PERMISSIONS_CHARGE + 88;
+const LIST_MUTABLE_DATA_PERMISSIONS_CHARGE: u64 = (500 * 44) + 40;
+const LIST_AUTH_KEYS_AND_VERSION_CHARGE: u64 = (500 * 32) + 44;
+const GET_ACCOUNT_INFO_CHARGE: u64 = 48;
```

Comment:
just thinking whether this small charge (48) may allow some malicious guy sends out bunch of small get_account req to choke the proxy by number of messages.

also, LIST_AUTH_KEYS_AND_VERSION_CHARGE is about 16k, LIST_MUTABLE_DATA_PERMISSIONS_CHARGE and GET_MUTABLE_DATA_SHELL_CHARGE are both around 22k.
and they are all just based on 500 entries (is it a hard limit or just an assumption).

can we combine these four into one unified charge, say 22k ? and put a comment to explain (assumption of around 500 entries)?